kolibri/plugins/coach/assets/src/views/common/resourceSelection/UpdatedResourceSelection.vue

```diff
@@ -235,6 +245,9 @@
         }
         return ViewMoreButtonStates.NO_MORE;
       },
+      showRadioButton() {
+        return this.target === SelectionTarget.QUIZ;
```

Comment:
Using SelectionTarget here is a great idea - but I think we might need to extend it (see the comment about re: practice quizzes, which might be easier to just talk through " in person")